src/uwtools/templater.py

```diff
@@ -83,22 +83,22 @@ def parse_args(argv):
 
 def set_template(argv):
     '''Main section for rendering and writing a template file'''
-    logfile = os.path.join(os.path.dirname(__file__), "templater.log")
-    log = Logger(level='info', logfile_path=logfile)
-
-
     user_args = parse_args(argv)
 
+    logfile = os.path.join(os.path.dirname(__file__), ".logtemplater")
+    log = Logger(level='info', logfile_path=logfile)
     if user_args.verbose:
         log = Logger(level='debug', logfile_path=logfile, colored_log=True)
+        log.debug('Finished setting up debug file logging in %(logfile)s')
     elif user_args.quiet:
         log.propagate = False
 
-    log.info("Running script templater.py with args:n", f"{('-' * 70)}\n{('-' * 70)}")
+
+    log.info('Running script templater.py with args:n '+('-' * 70)+'\n'+('-' * 70))
     for name, val in user_args.__dict__.items():
         if name not in ["config"]:
-            log.info(f"{name:>15s}: {val}")
-    log.info(f"{('-' * 70)}\n{('-' * 70)}")
+            log.info(format(name).rjust(15) + ': ' + str.format(val))
```

Comment:
So I wasn't quite sure if you were having problems with f-strings (apologies if I missed that), but are you able to use formatting like this?
```
"{name:>15s}: {val}".format(name=name, val=val)
```

I'd expect that should render a string first, then pass it to the formatter as string without additional need to process.

Reply:
The issue was that logging predates fstrings and there's some internal dev concerns about it being a security vulnerability so its never been updated. Hmm, maybe, I was trying to avoid repeating variables while maintaining the fixed format, but I'll see if it'll let me get away with it.

Reply:
OK  confirmed - if this is the preferred syntax, I'll swap it over!